### DISCUSSION TITLE
[IDEA] Filter log traffic

### DISCUSSION THREAD

**Discussion by tixmcprods** [dannagle/PacketSender#272] [Ideas]
Hello,
For bidirectionnal devices, possible to filter log traffic, for example filter only incoming packets (not outgoing), or filter by contained text in ASCII.
Thanks !
PS : great program, so useful :)

**Comment by dannagle** [comment]
Thanks, I am happy you like Packet Sender. Please share it with your friends and coworkers. The more users it has, the more time I can get to improve it. Also, thank you for submitting an idea.

I'm hesitant to add another UI widget to the main window. I think it is already pretty busy. Instead, I thought of making the current search box, the one that is already searching your packets, also search the traffic log. It'll then be a dual-purpose box. I think that could work.

**Comment by tixmcprods** [comment]
Hello,
Thanks for reply.
I thought about filter as it has to be refreshed at each received packet. My project is under stress (waiting for macro support for intense) and I'd like to see only received ones, and ones containing "ERROR" inside.
Yes your main window is already full, perhaps a NDI subwindow for traffic could make main one lighten.
Best regards.
